CameraOverlay/switch_demo.py

```diff
@@ -34,7 +34,7 @@ def turn_off(led):
 			sleep(0.25)
 		prev_switch_state = switch_state
         # TODO: Remove hard coding of directory of python script
```

Comment:
I think we should implement the TODO :)

Reply:
I think so too. Happy to accept this PR for now so we can start cleanup of code.